CHANGELOG.md

```diff
@@ -17,4 +17,13 @@ Minor update mainly for the new scada hostname.
 ## 0.2.0 (October 28 2022)
 
 ### Changed
-- Updated `net-rpc-msgpackrp` to `v2` to address a diamond dependency problem for projects importing hcp-scada-provider and an older version of hashicorp/go-msgpack
\ No newline at end of file
+- Updated `net-rpc-msgpackrp` to `v2` to address a diamond dependency problem for projects importing hcp-scada-provider and an older version of hashicorp/go-msgpack
+
+## 0.2.1 (January 11 2023)
```

Comment:
Do you want to add the latest one on top followed by other versions below? See consul as an example: https://github.com/hashicorp/consul/blob/main/CHANGELOG.md